### DISCUSSION TITLE
v2/zh/cookbook/tutorial/content

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#366] [Announcements]
# v2/zh/cookbook/tutorial/content

本教程指引你如何在 VuePress 项目中控制内容生成。

https://vuepress-theme-hope.github.io/v2/zh/cookbook/tutorial/content.html

**Comment by itwanger** [comment]
README.md 区分大小写不？比如说 readme.md

**Reply by Mister-Hope** [reply]
区分，因为linux下可以同时有多个大小写的readme.md，可能引发奇异的后果。
